cluster/CHAPTER.rst

```diff
@@ -200,24 +199,22 @@ Furthermore, we need some way to initialize a new cluster.
 
 But enough talk of theory and algorithms -- let's have a look at the code.
 
-Introducing Cluster
-===================
+## Introducing Cluster
 
-The "Cluster" library in this chapter implements a simple form of Multi-Paxos.
+The *Cluster* library in this chapter implements a simple form of Multi-Paxos.
 It is designed as a library to provide a consensus service to a larger application.
 
-Users of this library will depend on its correctness, so it's important to structure the code so that we can see -- and test -- its correspondance to the specification.
-Complex protocols can exhibit complex failures, too, so we will build support for reproducing and debugging rare failures.
+Users of this library will depend on its correctness, so it's important to structure the code so that we can see -- and test -- its correspondence to the specification.
+Complex protocols can exhibit complex failures, so we will build support for reproducing and debugging rare failures.
 
 The implementation in this chapter is proof-of-concept code: enough to demonstrate that the core concept is practical, but without all of the mundane equipment required for use in production.
-However, the code is structured so that such equipment can be added later with minimal changes to the core implementation.
+The code is structured so that such equipment can be added later with minimal changes to the core implementation.
 
 Let's get started.
 
-Types and Constants
--------------------
+### Types and Constants
 
-Cluster's protocol uses 15 different message types, each defined as a Python ``namedtuple``.
+Cluster's protocol uses fifteen different message types, each defined as a Python ``namedtuple``.
 
 {{{ from_to cluster.py '# message types' '^$'
```

Comment:
Are you looking at the {{{ specifically?  If so, that can get stripped -- it's part of my system for including the actual code in the chapter, and the code isn't changing anymore.